README.md

```diff
@@ -6,46 +6,217 @@
 
 ![](src/main/resources/META-INF/pluginIcon.svg)
 
-This is a port of [git-machete](https://github.com/VirtusLab/git-machete#git-machete) into an IntelliJ plugin.
+Git Machete plugin is a robust tool that simplifies your git related workflow.
+It's a great complement of IntelliJ version control system.
```

Comment:
`a great complement to the IntelliJ version control system`

Reply:
or better to `JetBrains products' built-in version control system`